quickwit/quickwit-common/src/tower/rate.rs

```diff
@@ -53,6 +53,11 @@ impl ConstantRate {
         let work = bytes.as_u64();
         Self::new(work, period)
     }
+
+    pub fn bytes_per_sec(bytes: ByteSize) -> Self {
+        let work = bytes.as_u64();
```

Comment:
On line 46. Why do we force work to be >0?

Reply:
No reason I can recall. It did cause me an issue when trying to test the rate limiter... I'll fix it.